openfaas/values.yaml

```diff
@@ -50,3 +50,47 @@ ingress:
     kubernetes.io/ingress.class: nginx
   tls:
     # Secrets must be manually created in the namespace.
+resources:
+  enabled: false
+  nats:
+    requests:
+      memory: "32MiB"
+      cpu: "50m"
+    limits:
+      memory: "32MiB"
+      cpu: "50m"
+  queueworker:
+    requests:
+      memory: "32MiB"
+      cpu: "50m"
+    limits:
+      memory: "32MiB"
+      cpu: "50m"
+  prometheus:
+    requests:
+      memory: "32MiB"
```

Comment:
Needs 500MB RAM according to @stefanprodan.

Reply:
This is the RAM usage after 12h on GKE:
<img width="535" alt="openfaas" src="https://user-images.githubusercontent.com/3797675/32839065-058ec6be-ca0b-11e7-9eb8-21cf43ae7693.png">